src/main.go

```diff
@@ -306,6 +308,29 @@ func main() {
 		os.Exit(0)
 	}()
 	log.Infoln("Bench-routes is up and running")
+	url := fmt.Sprintf("http://localhost%s", port)
+	go func(url string) {
+		var err error
+		log_message := fmt.Sprintf("%s is running bench-routes at %s", browserName, url)
+
+		switch runtime.GOOS {
+		case "linux":
+			err = exec.Command(browserName, url).Start()
```

Comment:
I am sorry, but I feel the browser types, as which browser opens is trivial for the project. This is just my opinion, but other maintainers can argue.

Reply:
Certainly, it is more of a nice to have feature than need to have. But I was looking at it from the perspective of a command line only application, where for example you would be able to run bench routes in CLI mode by default, and have to pass an argument like `--web=firefox` for example in order to start it in the browser.

But then, I think opening any browser at all by default should be a natural progression from having to type it in yourself.
I would like to listen to other opinions as well, from anyone who uses the application as to how they would prefer it.